src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -88,9 +88,12 @@
 public abstract class AbstractAggregatorExecutor
     implements ComponentExecutor<OperationModel>, Lifecycle {
 
+  public static final String QUORUM_EXCEPTION = "QuorumException";
   final Logger LOGGER = LoggerFactory.getLogger(getClass());
   private static final String AGGREGATORS_MODULE_KEY = "AGGREGATORS";
 
+  private static final boolean failOnStartIfNoQuorum =
```

Comment:
Question about this, what about the possibility to set this as a param in the advanced config? It could be easier to configure for the users from the mule app instead of the environment.